NEWS.md

```diff
@@ -21,6 +21,9 @@
     nafill(x, fill=3.14)              # warns that precision has been lost
     nafill(x, fill=as.integer(3.14))  # no warning; the as.<type> conveys intent
     ```
+    
+2. `mean(na.rm=TRUE)` by group will now fully use _GForce_ optimization, [#4849](https://github.com/Rdatatable/data.table/issues/4849). Thanks to [h2oai/db-benchmark](https://github.com/h2oai/db-benchmark) project for spotting this issue.
```

Comment:
What's the word 'fully' for please?  Can that word just be removed?  If not, what was partial about using GForce optimization before?